src/ansys/mapdl/core/_commands/graphics/labeling.py

```diff
@@ -0,0 +1,1247 @@
+        range to minimum value (or range to maximum if minimum = 0) is less than 0.001.
+
+        Parameters
+        ----------
+        wn : str
+            Window number (or ALL) to which command applies (defaults to 1).
+
+        ncont : str
+            Number of contour values. ``NCONT`` defaults to 9 for X11 or WIN32 and to 128 for X11c or
+            WIN32C. The default graphics window display for 3D devices is a smooth continuous shading effect
+            that spans the maximum of 128 contours available. Use the :ref:`dv3d` command to create defined
+            banding for your contour values (values of 9 and 128 are displayed in smooth shading only). The
+            legend, however, will display only nine color boxes, which span the full range of colors
+            displayed in the graphics window.
+
+        vmin : str
+            Minimum contour value. If ``VMIN`` = AUTO, automatically calculate contour values based upon
+            ``NCONT`` uniformly spaced values over the min-max extreme range. Or, if ``VMIN`` = USER, set
+            contour values to those of the last display (useful when last display automatically calculated
+            contours).
+
+        vinc : str
+            Value increment (positive) between contour values. Defaults to ( ``VMAX`` - ``VMIN`` )
+            ``/NCONT``.
+
+        vmax : str
+            Maximum contour value. Ignored if both ``VMIN`` and ``VINC`` are specified.
+
+        Notes
+        -----
+
+        .. _s-CONTOUR_notes:
+
+        Values represent contour lines in vector mode, and the algebraic maximum of contour bands in raster
+        mode.
+
+        Regardless of how many contours ( ``NCONT`` ) are specified, the actual number of contours appearing
+        on your display depends also on the device name, whether the display is directed to the screen or to
+        a file, the display mode (vector or raster), and the number of color planes. (All of those items are
+        controlled via :ref:`show`.) In any case, regardless of whether they are smoothed or banded, only
+        128 contours can be displayed.
+
+        For more information about changing the number of contours, see `Creating Geometric Results Displays
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_bas/Hlp_G_BAS12_6.html>`_.
+
+        When the current Mechanical APDL graphics are not displayed as Multi-Plots:
+
+        * If the current device is a 3D device ( :ref:`show`,3D), the model contours in all active windows
+          are the same, even if separate :ref:`contour` commands are issued for each active window.
+        * Mechanical APDL maintains a single data structure (segment) containing one set of contours. The
+        program
+          displays the same segment in all windows. The view settings of each window constitute the only
+          differences in the contour plots in the active windows.
```

Comment:
```suggestion
        * Mechanical APDL maintains a single data structure (segment) containing one set of contours. The
          program
          displays the same segment in all windows. The view settings of each window constitute the only
          differences in the contour plots in the active windows.
```

Reply:
This error has been added to the related [issue](https://github.com/ansys/pyconverter-xml2py/issues/413)